internal/locald/sandboxmanager/sandbox_controller.go

```diff
@@ -43,6 +43,8 @@ func newSBController(log *slog.Logger, sandbox *tunapiv1.WatchLocalSandboxesResp
 	}
 	// run the controller
 	go ctrl.run()
+	// trigger a reconcile
+	ctrl.triggerReconcile()
```

Comment:
I would put this in `ctrl.run()` so it is more self-contained.

Reply:
I addressed this in https://github.com/signadot/cli/pull/94